docs/contributing.md

```diff
@@ -1,4 +1,6 @@
-<!-- The content of this file is manually kept in sync with ../CONTRIBUTING.md. There are formatting differences because this file uses Hugo templating. -->
```

Comment:
With a single line comment this hit a known Hugo bug: https://github.com/gohugoio/hugo/issues/7092.

Splitting to more than a single line seems to work.